### PR TITLE
feat(Simkl): add presence

### PR DESCRIPTION
**Pull request by Kuriel23**
## Description 
Add presence for Simkl. The presence shows what movies, lists, profiles, anime, and TV shows users are currently viewing and searching for information about.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image
image
image
image
image

</details>

### CODE REVIEW

File: websites/S/Simkl/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.11",
+	"apiVersion": 1,
+	"author": {
+		"id": "354233941550694400",
+		"name": "kuriel."
+	},
+	"service": "Simkl",
+	"description": {
+		"en": "Your All-in-One TV, Anime and Movie Tracking Platform"
```
Comment (darkvillager2) [inline websites/S/Simkl/metadata.json:10]: Official description
```suggestion
		"en": "Track what you watch. Discover new shows and movies. Receive episode notifications. Get personalized recommendations. Join Simkl for free."
```

Comment (darkvillager2) [inline websites/S/Simkl/metadata.json:10]: Official description
```suggestion
		"en": "Track what you watch. Discover new shows and movies. Receive episode notifications. Get personalized recommendations. Join Simkl for free."
```

File: websites/S/Simkl/presence.ts [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+						presenceData.state = textContent(".simkltvcountdownsubtitle");
+						presenceData.largeImageKey = getImage(
+							".simkltvcountdowninfoleftiage"
+						);
+					} else if (path[4] && path[4].startsWith("review")) {
+						presenceData.details = `Reading a ${textContent(
+							"#global_div > div.SimklHeaderBgShaddow > div > div > div.SimklTVReviewHeaderBottom > table > tbody > tr > td > table > tbody > tr > td:nth-child(2) > table > tbody > tr:nth-child(1) > td"
+						)}`;
+						presenceData.state = textContent(
+							"#global_div > div.SimklHeaderBgShaddow > div > div > div.SimklTVReviewHeaderBottom > table > tbody > tr > td > table > tbody > tr > td:nth-child(2) > table > tbody > tr:nth-child(2) > td"
```
Comment (darkvillager2) [inline websites/S/Simkl/presence.ts:68]: try to shorten selectors where possible.
```suggestion
							"span.ajLinkInside"
```

File: websites/S/Simkl/presence.ts [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+}
+
+function getImage(tags: string) {
+	return document.querySelector<HTMLImageElement>(tags)?.src;
+}
+
+presence.on("UpdateData", async () => {
+	const { pathname, search } = document.location,
+		path = pathname.split("/"),
+		presenceData: PresenceData = {
```
Comment (darkvillager2) [inline websites/S/Simkl/presence.ts:17]:
```suggestion
		presenceData: PresenceData = {
		largeImageKey: Assets.Logo,
```

File: websites/S/Simkl/presence.ts [hunk 4]
```diff
@@ -0,0 +1,374 @@
+const presence = new Presence({
+		clientId: "1290874564582051855",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
```
Comment (darkvillager2) [inline websites/S/Simkl/presence.ts:5]:
```suggestion

const enum Assets {
Logo =  "https://i.imgur.com/1m6HokC.png",
}
```

File: websites/S/Simkl/presence.ts [hunk 5]
```diff
@@ -0,0 +1,374 @@
...
+function textContent(tags: string) {
+	return document.querySelector(tags)?.textContent?.trim();
+}
+
+function getImage(tags: string) {
+	return document.querySelector<HTMLImageElement>(tags)?.src;
```
Comment (darkvillager2) [inline websites/S/Simkl/presence.ts:11]:
```suggestion
	return document.querySelector<HTMLImageElement>(tags)?.src ?? Assets.Logo;
```

**Review by darkvillager2 — CHANGES_REQUESTED** [review]

**Comment by Kuriel23** [comment]
Done.

**Review by theusaf — DISMISSED** [review]

**Comment by darkvillager2** [comment]
@theusaf there is no largeimagekey?

**Review by Bas950 — APPROVED** [review]

**Comment by Bas950** [comment]
> @theusaf there is no largeimagekey?

there is?

**Comment by darkvillager2** [comment]
> > @theusaf there is no largeimagekey?
> 
> there is?

Woops i was viewing an old commit xd

**Pull request merged**